### PR TITLE
[assertj] Added ServiceEventAssert and FrameworkEventAssert

### PR DESCRIPTION
**Pull request by kriegfrj**
Signed-off-by: Fr Jeremy Krieg <fr.jkrieg@greekwelfaresa.org.au>

### CODE REVIEW

File: org.osgi.test.assertj/src/main/java/org/osgi/test/assertj/event/AbstractBitmappedTypeEventAssert.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+		return getType.applyAsInt(actual);
+	}
+
+	public SELF isOfType(int expected) {
+		isNotNull();
+		if ((expected & (expected - 1)) != 0) {
+			throw new IllegalArgumentException(
+				"Multiple bits set in expected (" + expected + ") - do you mean to use isOfTypeMaskedBy()?");
+		}
+		final String expectedString = bitmap.toString(expected);
```
Comment (bjhargrave) [inline org.osgi.test.assertj/src/main/java/org/osgi/test/assertj/event/AbstractBitmappedTypeEventAssert.java:47]: We don't need the expectedString unless there is a failure, right? So move it inside the if.

File: org.osgi.test.assertj/src/main/java/org/osgi/test/assertj/event/AbstractBitmappedTypeEventAssert.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+		if ((actualType() & expected) == 0) {
+			failWithMessage("%nExpecting%n <%s>%nto be of type:%n <%d:%s>%n but was of type:%n <%s>", actual, expected,
+				expectedString, bitmap.maskToString(actualType()));
+		}
+		return myself;
+	}
+
+	public SELF isNotOfType(int expected) {
+		isNotNull();
+		final String expectedType = bitmap.toString(expected);
```
Comment (bjhargrave) [inline org.osgi.test.assertj/src/main/java/org/osgi/test/assertj/event/AbstractBitmappedTypeEventAssert.java:57]: We don't need the expectedString unless there is a failure, right? So move it inside the if.

File: org.osgi.test.assertj/src/main/java/org/osgi/test/assertj/event/AbstractBitmappedTypeEventAssert.java (hunk 1 above)
Comment (kriegfrj) [reply org.osgi.test.assertj/src/main/java/org/osgi/test/assertj/event/AbstractBitmappedTypeEventAssert.java]: Done.

File: org.osgi.test.assertj/src/main/java/org/osgi/test/assertj/event/AbstractBitmappedTypeEventAssert.java (hunk 2 above)
Comment (kriegfrj) [reply org.osgi.test.assertj/src/main/java/org/osgi/test/assertj/event/AbstractBitmappedTypeEventAssert.java]: Done.

**Comment by kriegfrj** [comment]
Thanks for the review!

**Pull request merged**